src/components/Registration/BusinessTypeConfirm.vue

```diff
@@ -105,21 +113,26 @@ export default class BusinessTypeConfirm extends Vue {
   }
 }
 
-#business-check-div {
-  display: flex;
+#business-checkbox-div {
   margin-top: 20px;
 }
 
-#business-check-div .mt-0 {
-  margin-left: -3px;
-  padding-top: 1px;
+.certify-stmt {
+  display: inline;
+  font-size: 0.875rem;
+  color: $gray7;
+  font-weight: normal;
 }
 
 ::v-deep .v-input--checkbox .v-input__slot {
   align-items: flex-start;
 }
 
-.invalid-section ::v-deep .v-input--checkbox .v-input__slot .v-label {
+::v-deep .v-input--checkbox .v-input__slot .v-input--selection-controls__input {
```

Comment:
Please add comment saying what you're overriding here. Also on line 127 above.

Reply:
This css is to align the checkbox to the very left of text above: margin-left: -3px. Otherwise, the checkbox is shifted by 3px to the right.

Reply:
Code comment, please.

Reply:
For sure. I added the comments in my last commit: ced942f.

Reply:
Great comment, thanks!